svc/pkg/linode/standalone/gc/src/lib.rs

```diff
@@ -40,7 +40,8 @@ pub async fn run_from_env(pools: rivet_pools::Pools) -> GlobalResult<()> {
 	);
 
 	// Build HTTP client
-	let client = util_linode::Client::new_with_headers(headers).await?;
+	let api_token = util::env::read_secret(&["linode", "token"]).await?;
```

Comment:
Won't this only run for the main cluster? Don't we need to iterate over all clusters?